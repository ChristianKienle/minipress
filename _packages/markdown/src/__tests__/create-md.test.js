// @ts-check
const createMd = require('./../create-md')

describe('md', () => {
  it('works', () => {
    const md = createMd()
    const markup = md.render('# hello <MyBadge />')
    expect(markup).toMatchSnapshot()
  })

  it('works2', () => {
    const md = createMd()
    const text = '# title\n\ntest `cool` test <MyButton /> bla\n\n'
    const markup = md.render(text)
    expect(markup).toMatchSnapshot()
  })
})