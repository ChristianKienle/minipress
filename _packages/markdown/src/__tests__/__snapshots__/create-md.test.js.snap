// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`md works 1`] = `
"<h1>hello <MyBadge /></h1>
"
`;

exports[`md works2 1`] = `
"<h1>title</h1>
<p>test <code>cool</code> test <MyButton /> bla</p>
"
`;
